src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageFormDetails.tsx

```diff
@@ -64,9 +65,12 @@ const WorkPackageEditDetails: React.FC<Props> = ({
   );
 
   return (
-    <PageBlock title="Work Package Details">
+    <Box>
+      <Typography variant="h5" sx={{ marginBottom: '10px', color: 'white' }}>
+        Project Details
+      </Typography>
       <Grid container xs={12}>
```

Comment:
you can put spacing={1} here and remove all the sx={mt, mr} below

Reply:
I tried putting spacing in but it seems to not be a property of Typography and could you clarify what you mean when you say to remove all the sx={mt, mr}

Reply:
put it on the grid, not the container

Reply:
when I say remove that sx prop, I'm referring to where it's used on all the grid items below